examples/ring_app/logback.xml

```diff
@@ -0,0 +1,15 @@
+<configuration scan="true">
+    <appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
+        <encoder>
+            <pattern>%d %-5p [%c{2}] %m%n</pattern>
+        </encoder>
+    </appender>
+
+    <root level="info">
+        <appender-ref ref="STDOUT" />
+    </root>
+
+    <!-- Uncomment the following loggers to turn on rule-by-rule logging -->
+    <!-- <logger name="puppetlabs.trapperkeeper.authorization.rules" level="TRACE"/> -->
+    <!-- <logger name="puppetlabs.trapperkeeper.services.authorization.authorization-service" level="DEBUG"/> -->
```

Comment:
d'oh, yeah, you're right.  Copypasta from the ring example I lifted from the trapperkeeper-authorization code.  I removed that in df5c156.